dev/.rat-excludes

```diff
@@ -4,8 +4,6 @@ cache
 .gitattributes
 .project
 .classpath
-.mima-excludes
```

Comment:
After this PR https://github.com/apache/spark/commit/d17d221487fa7a3af6f4af2217f1d4889ceb084d, the `.mima-excludes` file has become invalid.